packages/devtools_app/lib/src/inspector/flutter/layout_tab.dart

```diff
@@ -0,0 +1,190 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+import '../../inspector/inspector_text_styles.dart' as inspector_text_styles;
+import '../diagnostics_node.dart';
+import '../inspector_tree.dart';
+
+class InspectorDetailsTabController extends StatelessWidget {
+  const InspectorDetailsTabController({this.detailsTree, Key key})
+    : super(key: key);
+
+  final Widget detailsTree;
+
+  @override
+  Widget build(BuildContext context) {
+    return DefaultTabController(
+      length: 2,
+      child: Column(
+        children: <Widget>[
+          const TabBar(
+            tabs: [
+              Tab(text: 'Details Tree'),
+              Tab(text: 'Layout Details'),
+            ],
+          ),
+          Expanded(
+            child: TabBarView(
+              children: [
+                detailsTree,
+                Container(),
+              ],
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+// TODO(albertusangga): Remove this linter ignore
+// ignore: must_be_immutable
+class StoryOfYourFlexWidget extends StatelessWidget {
+  StoryOfYourFlexWidget(this.node, {
+    Key key,
+  }) : super(key: key) {
+    properties = FlexProperties.fromJson(node.diagnostic.flexDetails);
+  }
+
+  final InspectorTreeNode node;
+
+  // Deserialized information about Flex elements
+  FlexProperties properties;
+
+  @override
+  Widget build(BuildContext context) {
+    final List<Widget> children = [
+      for (RemoteDiagnosticsNode child in node.diagnostic.childrenNow)
+        Expanded(
+          flex: 1,
+          child: Container(
+            decoration: BoxDecoration(
+              color: Theme
+                .of(context)
```

Comment:
Theme.of(context) is repeated a few times. Cache it as a local variable at the top of this method.